src/Audio.md

```diff
@@ -0,0 +1,110 @@
+Channel 3, the "wave" channel, produces arbitrary user-supplied waves.
+Channel 4 is the "[noise](https://en.wikipedia.org/wiki/Noise_in_music)" channel, producing a pseudo-random wave.
+
+The VIN channel is a signal received directly from the cartridge, allowing external hardware to supply a fifth sound channel.
+No licensed games used this feature, and it was omitted from the Game Boy Advance.
+
+::: tip POCKET MUSIC
+
+Despite some online claims, *Pocket Music* does not use VIN.
+It refuses to run on the GBA for a different reason: the developer couldn't figure out how to silence buzzing associated with sample playback on the wave channel.
+
+:::
+
+[^speaker_mono]:
+The speaker merges back the two channels, losing the stereo aspect entirely.
+
+## Common concepts
+
+### APU
+
+The Game Boy's sound chip is called the <abbr title="Audio Processing Unit">APU</abbr>.
+
+The APU runs off the same master clock as the rest of the Game Boy, which is to say, it is fully synced with the CPU and [PPU](<#Rendering Overview>).
+This also means that the APU runs about 2.4% faster on the SGB1, increasing frequencies by as much and thus sounding slightly higher-pitched.
+The SGB2 rectifies this issue.
+
+The APU processes a lot of events at a fixed rate dependent on the event's type.
+All interfaces to the APU use **durations** instead of frequencies, which may be confusing as signal theory and music are more typically based on the latter.
+Thus, durations will be expressed from their frequencies: for example, a "256 Hz tick" means "1 ∕ 256th of a second".
+
+The length of APU ticks is not affected by [CGB double speed](<#FF4D — KEY1 (CGB Mode only): Prepare speed switch>), so the APU works just the same regardless of CPU speed.
+
+::: warning
+
+The Game Boy's APU is actually full of tricky details; this chapter will mostly describe the intended / common behavior, and often paper over bugs & quirks.
+Readers wishing to learn more should read the [APU details](<#Audio Details>) chapter.
+
+:::
+
+### Triggering
+
+**Triggering** a channel causes it to turn on if it wasn't[^trig_dac_off], and to start playing its wave from the beginning[^pulse_restart].
+Most changes to a channel's parameters take effect immediately, but some require re-triggering the channel.
+
+### Volume & envelope
+
+The volume can be controlled in two ways: there is a "master volume" control[^vol_knob] (which has separate settings for the left and right outputs), and each channel's volume can be individually set as well (CH3's less precisely than the others).
+
+Additionally, an [**envelope**](https://en.wikipedia.org/wiki/Envelope_(music)) can be configured for all channels except CH3, which allows automatically adjusting the volume over time.
+The parameters that can be controlled are how many times the envelope will be stepped, in which direction, and how long between steps.
```

Comment:
Too heavy on technical primitives. I'd describe this as something like:
> The envelope register allows setting the initial volume, the envelope direction and envelope speed.

Reply:
I'd avoid the word "register" here too, then:
```suggestion
The parameters that can be controlled are the initial volume, the envelope's direction, and its speed.
```

Reply:
Register is fine, since that's the way the programmer interfaces with the hardware, and something you need to learn early to do anything with the hardware. What I really mean by technical primitives, is _hidden_ technical primitives. Ie, new extra concepts you have to understand, just to read a basic sentence. With that said, there's nothing wrong with the suggested change, per se.

Reply:
I'm generally aiming for `Audio.md` to be readable by non-programmers, `Audio_Registers.md` by programmers, and `Audio_details.md` by people who really need to dive into some of the details (programmers trying to understand/debug weird behaviour, emu devs, etc.)